tests/printer.test.ts

```diff
@@ -53,18 +53,27 @@ test("Conditions", () => {
       operator: Operator.eq,
       value: 20
     },
-    "cs.{example,new}": {
+    "cs.(example,new)": {
```

Comment:
@msxrx is this accepted by PostgREST as well?

Reply:
yes